### ISSUE TITLE
Question on landmark and dataset

### ISSUE THREAD

**Issue by nicenice97** [fengju514/Face-Pose-Net#20]
Your paper's name is "landmark-free", I want to know if there is a step to extract landmarks?
Is the landmark extracted but not displayed, or is the landmark not extracted at all?

By the way, in your paper, you said the training set is produced by running on existing landmark detector on VGG dataset. Are all 2.6 million images used for training?

**Comment by nicenice97** [comment]
I still want to know what is the true database for your machine learning. Thank you.

**Comment by fengju514** [comment]
To get the 3D head pose labeled training data, we did extract landmarks for that purpose.

Regarding to the question of "Are all 2.6 million images used for training?"
Yes, but we saw the validation accuracy saturates with less than 2.6 million training image as well. This may also attribute to that we did the perturbations on each image.

**Comment by nicenice97** [comment]
Thank you! So the database that is really used for training is the pose labels of human face. Does this training data exist in PAM_frontal_AlenNet.npy? 
How many pose labels are used to train it? The time required to run your code is very short, so I don't think there are 2.6 million labels.

**Comment by cdwangjiang** [comment]
the model  downloading link doesn't work，could you share a new downloading link that works?i will appreciate your kindness very much!

**Comment by iacopomasi** [comment]
Please, share with us your email and will provide the link for the model. Thanks
